### DISCUSSION TITLE
Master File zip is corrutped.

### DISCUSSION THREAD

**Discussion by HyperXinon** [KadeArchive/Kade-Engine#532] [Q&A]
Im trying to download the master of kade engine but the file is broken, i tried it 2 times now which both give out the same error

![Problem 1](https://user-images.githubusercontent.com/62767605/119160837-f5e13e80-ba58-11eb-92da-b2bd57483d46.PNG)
![Problem 2](https://user-images.githubusercontent.com/62767605/119160841-f679d500-ba58-11eb-90d4-f066a61f1c1e.PNG)

On the Problem 2.png file the translation means:
This Folder cannot be opened.

The Zip-Compromised Folder "File Path" Is Invalid.

**Comment by shaderw0lf** [comment]
try to redownload or unzip with 7zip...??

**Reply by HyperXinon** [reply]
redownloading doesnt work,  i did it 3 times already, and 7ip or winrar isnt the issue as the file is corrupt before trying to extract with winrar.

**Reply by ghost** [reply]
Try using File Explorer. It works most of the time.

**Reply by HyperXinon** [reply]
in the first picture is a representation that file explorer doesnt work too :(

**Reply by HyperXinon** [reply]
oh wait, yeah. i pressed the wrong button for 7-zip. sorry fo rthe incovenience.

**Comment by puyoxyz** [comment]
Use git clone. GitHub zips suck.

**Reply by ghost** [reply]
yep
